concepticondata/conceptlists/Morucci-2019-643.tsv

```diff
@@ -0,0 +1,644 @@
+Morucci-2019-643-214	214	fragile	fragile	0.278	0.162	4	1.361	4.351	4.351	vista	10.152	1.164	0.413	6.278	3.55	6.421		
+Morucci-2019-643-215	215	fragrante	fragrant	4.412	2.853	2.088	1.529	2.941	4.412	gusto	13.824	1.549	0.209	4.474	5.176	5.368	376	FRAGRANT
+Morucci-2019-643-216	216	freddo	cold	3.189	0.541	4.919	0.629	2.838	4.919	tatto	12.115	1.35	0.361	6.941	2.278	6.833	1287	COLD
+Morucci-2019-643-217	217	frenetico	frenetic	0.128	0.154	0.487	3	4.103	4.103	vista	7.872	1.023	0.505	6.118	5	5.333		
+Morucci-2019-643-218	218	fresco	fresh	4	2.529	4.324	0.706	3.324	4.324	tatto	14.882	1.493	0.243	6.882	2.944	6.556	243	FRESH
+Morucci-2019-643-219	219	friabile	brittle	2.639	0.622	4.162	1.919	3.622	4.162	tatto	12.963	1.474	0.273	4.316	4.824	5.632		
+Morucci-2019-643-220	220	frignante	whining	0	0.077	0.053	4.615	3.256	4.615	udito	8.001	0.761	0.577	3.111	4.789	5		
+Morucci-2019-643-221	221	frusciante	rustling	0.152	0.147	1.176	4.853	2.147	4.853	udito	8.475	1.083	0.555	4.5	5.526	4.316		
+Morucci-2019-643-222	222	fruttato	fruity	4.7	4.275	0.615	0.154	2.575	4.7	gusto	12.319	1.267	0.369	5.118	4.056	5.722		
+Morucci-2019-643-223	223	fruttoso	fruity	4.462	3.897	0.744	0.179	3.077	4.462	gusto	12.359	1.308	0.347	4.235	4.25	4.833		
+Morucci-2019-643-224	224	fucsia	fuchsia	0.103	0.368	0.026	0.103	5	5	vista	5.6	0.452	0.888	6.412	2.722	5.389		
+Morucci-2019-643-225	225	fulgido	shining	0.171	0.229	0.686	0.4	3.486	3.486	vista	4.971	0.983	0.667	3.235	6.286	2.556		
+Morucci-2019-643-226	226	fumoso	smoky	1.1	3.925	0.6	0.425	4.425	4.425	vista	10.475	1.262	0.382	4.824	4.706	4.5		
+Morucci-2019-643-227	227	gelato	ice cream	2.949	0.872	4.846	0.462	3.641	4.846	tatto	12.769	1.367	0.343	6.941	2.444	6.667		
+Morucci-2019-643-228	228	gelido	icy	2.154	0.825	4.875	0.725	3.05	4.875	tatto	11.629	1.389	0.357	6.529	3.667	6.333		
+Morucci-2019-643-229	229	generico	generic	0.769	0.795	1	2.026	2.846	2.846	vista	7.436	1.465	0.279	6.053	4.588	4.526		
+Morucci-2019-643-230	230	geniale	ingenious	0.818	0.485	0.848	3.485	3.091	3.485	udito	8.727	1.343	0.344	6.667	4.2	6		
+Morucci-2019-643-231	231	ghiacciato	icy	2.9	0.95	4.8	0.925	4.2	4.8	tatto	13.775	1.423	0.281	6.105	2.882	6.579		
+Morucci-2019-643-232	232	ghiaioso	gravelly	0.189	0.324	4.243	2.811	4.784	4.784	vista	12.351	1.231	0.372	3.588	4.059	4.944		
+Morucci-2019-643-233	233	giallastro	yellowish	0.385	0.308	0.154	0	4.974	4.974	vista	5.82	0.565	0.855	4.421	3.353	4.526		
+Morucci-2019-643-234	234	giallo	yellow	0.231	0.256	0.128	0.105	4.974	4.974	vista	5.695	0.547	0.855	6.737	2.294	6.684	1424	YELLOW
+Morucci-2019-643-235	235	giallognolo	yellowish	0.225	0.325	0.075	0.1	4.9	4.9	vista	5.625	0.543	0.858	4.158	4.294	3.842		
+Morucci-2019-643-236	236	gigante	giant	0.091	0.091	3.636	1.333	4.879	4.879	vista	10.03	1.072	0.477	6.389	2.6	6.421		
+Morucci-2019-643-237	237	giusto	right	1.175	1.05	1.1	2.5	2.925	2.925	vista	8.75	1.509	0.214	6.842	2.765	5.263	1019	RIGHT
```

Comment:
CORRECT (RIGHT)

Reply:
I mapped the concept already to _corretto_. Is _giusto_ more correct ;)

Reply:
You can really say they are synonyms, but "giusto" is more common. If we want to be picky about semantics, I'd say that (especially in higher diaphasic levels, Italians might still differentiate between "giusto" as what is right/fair (as in opposition to wrong), and "corretto" as what is corrected (i.e., not wrong anymore, adjusted, adequate), but honestly they are synonyms with "corretto" sounding a bit more formal.

Corretto can also be an espresso coffee with a shot of liquor, as it is "adjusted" :wink:

Reply:
OK :) I'll remap.